docs/releasehistory.rst

```diff
@@ -4,13 +4,17 @@ Release History
 0.2Y.Z - XXXXXX release
 =======================
 
-0.20.4 - Bugfix release
-======================
+0.21.0 - Bugfix release
+=======================
+
 
 Bugfixes
 --------
 - Fixes TestAbsoluteAlchemicalFactory.test_overlap NaNs (`#534 <https://github.com/choderalab/openmmtools/pull/534>`_)
 - Try closing reporter in test for windows fix (`#535 <https://github.com/choderalab/openmmtools/pull/535>`_) 
+- Follow NEP 29 and test newer python versions and drop old ones (`#542 <https://github.com/choderalab/openmmtools/pull/542>`_)
+- Update to handle the new OpenMM 7.6 package namespace (`#528 <https://github.com/choderalab/openmmtools/pull/528>`_)
+- Context cache fixes (`#524 <https://github.com/choderalab/openmmtools/pull/524>`_) & (`#538 <https://github.com/choderalab/openmmtools/pull/538>_`)
```

Comment:
Thanks for adding these to the docs. In this case #524 was superseded by #538 , so maybe just mentioning #538 is enough, since the `deepcopy` behavior is no longer needed with the new context cache cleanup. This is non-blocking of course.

```suggestion
- Context cache usage cleanup (`#538 <https://github.com/choderalab/openmmtools/pull/538>_`). Avoiding memory issues and more streamlined API usage of `ContextCache` objects.
```

Reply:
good idea!